src/main/java/org/zalando/nakadi/filters/LoggingFilter.java

```diff
@@ -22,10 +22,12 @@
 import javax.servlet.http.HttpServletResponse;
 import java.io.IOException;
 import java.util.Optional;
+import java.util.regex.Pattern;
 
 @Component
 public class LoggingFilter extends OncePerRequestFilter {
 
+    private static final Pattern PATTERN_POST_PUBLISH = Pattern.compile("\\/event-types\\/.*\\/events");
```

Comment:
In general I agree with @v-stepanov comment https://github.com/zalando/nakadi/pull/1022/files#r254837222 
In case if you will decide to leave it, maybe you can enrich pattern with `^` and `$`